src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -129,18 +225,44 @@ const TeamsTools = () => {
               />
             </FormControl>
             <FormControl fullWidth>
-              <FormLabel>Description</FormLabel>
-              <ReactHookTextField
+              <Controller
                 name="description"
                 control={control}
-                fullWidth
-                multiline
-                rows={5}
-                errorMessage={errors.description}
-                maxLength={300}
+                rules={{ required: true }}
+                render={({ field }) =>
+                  isEditingDescription ? (
```

Comment:
Currently if isEditingDescription is false and the user tried to submit the error text is hidden. I don't think you need isEditingDescription at all, it should work if you just make it true all the time